docker/cami-store/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM mysql:5.6
+
+# Add the store schema to the container to be run when the first container is
+# run. This requires that the build context is the project root folder. Run
+# build like:
+# `docker build -f docker/cami-store/Dockerfile .`
+ADD store/cami.sql /docker-entrypoint-initdb.d/
+
+# Hardcode passwors until we learn a better way to do this.
```

Comment:
I think people will either use a service to distribute secrets (not sure how it's done in Docker right now), or hardcode the ENV vars in docker-compose.yml (this is what I've seen in some of our clients' stacks). We definitely need to dig up a better solution, I don't think hardcoding in docker-compose.yml is that good either.